rustls/examples/internal/bogo_shim.rs

```diff
@@ -2,1293 +2,11 @@
 // test suite. See bogo/ for this in action.
 //
 // https://boringssl.googlesource.com/boringssl/+/master/ssl/test
-//
```

Comment:
Would it be feasible to separate the `bogo_shim.rs` -> `bogo_shim_impl.rs` move in its own commit ahead of this work, maintaining the feature gates, and then removing those gates in this commit?

Reply:
Yes, will do that.